src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -772,6 +790,7 @@ private String fillUpStudentRejoinAfterGoogleIdResetFragment(StudentAttributes s
     }
 
     private String fillUpInstructorJoinFragment(InstructorAttributes instructor) {
+        // can this be abstracted into a function getInstructorJoinUrl() to avoid duplication
```

Comment:
Yes we should abstract out repeated code, just rmb to remove this comment after u have done so